### PR TITLE
Add Trino kustomize descriptors using upstream ODH

### PR DESCRIPTION
**Pull request by rimolive**

### CODE REVIEW

File: kfdefs/overlays/dev/kustomization.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - dh-dev-trino
```
Comment (HumairAK) [inline kfdefs/overlays/dev/kustomization.yaml:6]: this and many other files are lacking newlines at the end of files.

File: kfdefs/overlays/stage/dh-stage-trino/kustomization.yaml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+          kustomizeConfig:
+            repoRef:
+              name: idh
+              path: odh-manifests/trino
+          name: trino
+      - op: add
+        path: /spec/repos/-
+        value:
+          name: idh
+          uri: https://github.com/AICoE/internal-data-hub/tarball/main
```
Comment (HumairAK) [inline kfdefs/overlays/stage/dh-stage-trino/kustomization.yaml:23]: this patch seems to exist in all 3 environments, why not add it to base?

**Review by anishasthana — COMMENTED** [review]
/lgtm

**Review by accorvin — COMMENTED** [review]
/approve

**Pull request merged**